assets/javascripts/discourse/components/chat-composer.js

```diff
@@ -44,10 +44,7 @@ export default Component.extend(TextareaTextManipulation, {
   inProgressUploads: null,
   composerEventPrefix: "chat",
   composerFocusSelector: ".chat-composer-input",
-  canAttachUploads: or(
-    "siteSettings.chat_allow_uploads",
-    "chatChannel.isDirectMessageChannel"
-  ),
+  canAttachUploads: reads("siteSettings.chat_allow_uploads"),
```

Comment:
Just confirming we have a test that covers this?

Reply:
Good question, will check!

Reply:
Done